From 71425e6febbfdaf949f49f922cc194cf3d257060 Mon Sep 17 00:00:00 2001
From: Abinaya P <abinayap@codeaurora.org>
Date: Tue, 3 Jul 2018 22:33:45 +0530
Subject: [PATCH] ARM: asm/div64.h: include bug.h file to resolve compilation
 issue

Include bug.h file to resolve compilation issue while picking
latest do_div fixes to 3.18 kernel.

Change-Id: Iaaf0f72f603cdc0aa577842a023d68e77598821a
Signed-off-by: Abinaya P <abinayap@codeaurora.org>
---
 arch/arm/include/asm/div64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/include/asm/div64.h b/arch/arm/include/asm/div64.h
index e1f07764b0d6..23eca1828725 100644
--- a/arch/arm/include/asm/div64.h
+++ b/arch/arm/include/asm/div64.h
@@ -59,7 +59,7 @@ static inline uint32_t __div64_32(uint64_t *n, uint32_t base)
 #define do_div(n, base) __div64_32(&(n), base)
 
 #else
-
+#include <asm/bug.h>
 /*
  * gcc versions earlier than 4.0 are simply too problematic for the
  * __div64_const32() code in asm-generic/div64.h. First there is
-- 
2.17.0

